src/guides/v2.2/cloud/release-notes/mcd-release-notes.md

```diff
@@ -25,4 +25,6 @@ The release notes include:
 
 -  {:.new}Docker for Linux does not require a third party utility to support file syncing because binding a host directory to a container does not cause the same performance issues. Added the `--sync-enggine=native` option.<!--MAGECLOUD4351-->
 
--  {:.new}Added the `--rm` option to `./bin/magento-docker` commands for the build and deploy containers. This removes the container after the task is complete.<!--MAGECLOUD-4205-->
\ No newline at end of file
+-  {:.new}Added the `--rm` option to `./bin/magento-docker` commands for the build and deploy containers. This removes the container after the task is complete.<!--MAGECLOUD-4205-->
+
+-  {:.new}Added the `--sync-engine=native` option to the `docker-build` command to disable file synchronization when launching the Docker generator in developer mode. Use this option when developing on  Linux systems, which do not require file synchronization for local Docker development.<!--MAGECLOUD-4351-->
```

Comment:
Docker build options are listed in the Docker quick reference, too. So this is an opportunity to update that list.